app/routes/first-time/eligibility/claim/claim-summary.js

```diff
@@ -36,7 +36,7 @@ module.exports = function (router) {
     getClaimSummary(req.params.claimId)
       .then(function (claimDetails) {
         savedClaimDetails = claimDetails
-        var claimSummary = new ClaimSummary(claimDetails.claim.visitConfirmation) // eslint-disable-line no-unused-vars
+        var claimSummary = new ClaimSummary(claimDetails.claim.visitConfirmation, claimDetails.claim.Benefit, claimDetails.claim.benefitDocument[0]) // eslint-disable-line no-unused-vars
```

Comment:
add safety check to ensure `claimDetails.claim.benefitDocument` is not null/undefined/empty before calling `claimDetails.claim.benefitDocument[0]` which would throw an erorr